presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkModule.java

```diff
@@ -206,7 +206,7 @@ protected void setup(Binder binder)
 
         // json codecs
         jsonCodecBinder(binder).bindJsonCodec(ViewDefinition.class);
-        jsonCodecBinder(binder).bindJsonCodec(TaskStats.class);
+        jsonCodecBinder(binder).bindJsonCodec(TaskInfo.class);
```

Comment:
I think it's right to use `TaskInfo` instead of `TaskStats`. Just curious why previous `TaskStats` is chosen?

Reply:
> Just curious why previous TaskStats is chosen?

Just some premature coding. I was trying to prototype something, and hadn't finish =)